docs/predictive-analytics/next-best-offer/manual.sql

```diff
@@ -0,0 +1,24 @@
+-- Manually insert data if source connectors are not run
```

Comment:
This is a good comment but I'd like to look into doing this programmatically for all recipes (none of which have a comment in this file).  Let's please delete for now, for consistency.